electrum/commands.py

```diff
@@ -1322,6 +1322,34 @@ async def reverse_swap(self, lightning_amount, onchain_amount, wallet: Abstract_
             'onchain_amount': format_satoshis(onchain_amount_sat),
         }
 
+    @command('n')
+    async def convert_currency(self, from_amount, from_ccy = None, to_ccy = 'BTC'):
+        """Converts the given amount of currency to another using the
+        configured exchange rate source.
+        """
```

Comment:
I think both `from_ccy` and `to_ccy` should default to `fx.ccy` and "BTC", as appropriate.
```suggestion
    async def convert_currency(self, from_amount, from_ccy=None, to_ccy=None):
        if to_ccy is None:
            to_ccy = "BTC" if from_ccy != "BTC" else self.daemon.fx.ccy
        if from_ccy is None:
            from_ccy = "BTC" if to_ccy != "BTC" else self.daemon.fx.ccy
```
This would make the following queries easy:
```
$ ./run_electrum convert_currency 1000
{
    "from_amount": "1000.00",
    "from_ccy": "EUR",
    "source": "CoinGecko",
    "to_amount": "0.06193907",
    "to_ccy": "BTC"
}
$ ./run_electrum convert_currency 1 --from_ccy BTC
{
    "from_amount": "1.00000000",
    "from_ccy": "BTC",
    "source": "CoinGecko",
    "to_amount": "16144.90",
    "to_ccy": "EUR"
}
```
-----
Alternatively, we could switch around the defaults, so that `convert_currency` without arguments returns the current price:
```suggestion
    async def convert_currency(self, from_amount=1, from_ccy=None, to_ccy=None):
        if from_ccy is None:
            from_ccy = "BTC" if to_ccy != "BTC" else self.daemon.fx.ccy
        if to_ccy is None:
            to_ccy = "BTC" if from_ccy != "BTC" else self.daemon.fx.ccy
```
```
$ ./run_electrum convert_currency
{
    "from_amount": "1.00000000",
    "from_ccy": "BTC",
    "source": "CoinGecko",
    "to_amount": "16144.32",
    "to_ccy": "EUR"
}
$ ./run_electrum convert_currency --from_amount 1000 --from_ccy EUR
{
    "from_amount": "1000.00",
    "from_ccy": "EUR",
    "source": "CoinGecko",
    "to_amount": "0.06194129",
    "to_ccy": "BTC"
}
$ ./run_electrum convert_currency --from_amount 0.25 --from_ccy BTC
{
    "from_amount": "0.25000000",
    "from_ccy": "BTC",
    "source": "CoinGecko",
    "to_amount": "4036.08",
    "to_ccy": "EUR"
}
```
-----
Both choices look reasonable to me.